doc/mgr/dashboard.rst

```diff
@@ -94,6 +97,45 @@ a list like so::
 Configuration
 -------------
 
+SSL/TLS Support
+^^^^^^^^^^^^^^^
+
+All HTTP connections to the dashboard are secured with SSL/TLS. 
+
+You can generate a self-signed certificate with the command::
+
+  $ ceph dashboard create-self-signed-cert
+
+Note that most web browsers will complain about self-signed certificates and
+require explicit confirmation before establishing a secure connection to the
+dashboard.
+
+To properly secure a deployment, a certificate that is signed by your
+organization's certificate authority should be used.
+
+For example, a key pair can be generated with a command similar to::
+
+  $ openssl req -new -nodes -x509 \
+    -subj "/O=IT/CN=ceph-mgr-dashboard" -days 3650 \
+    -keyout dashboard.key -out dashboard.crt -extensions v3_ca
+
+The ``dashboard.crt`` file should then be signed by your organization's CA
+(certificate authority). Once that is done, you can set it with::
+
+  $ ceph config-key set mgr mgr/dashboard/$name/crt -i dashboard.crt
+  $ ceph config-key set mgr mgr/dashboard/$name/key -i dashboard.key
+
+where ``$name`` is the name of the ``ceph-mgr`` instance (usually the
+hostname). If all manager instances are to share the same certificate,
+you can leave off the ``$name`` portion::
+
+  $ ceph config-key set mgr mgr/dashboard/crt -i dashboard.crt
+  $ ceph config-key set mgr mgr/dashboard/key -i dashboard.key
+
+.. note::
+  You need to restart the Ceph manager processes after changing the
```

Comment:
This is only true if you run the above `config-key` commands after enabling the dashboard module.
Maybe we can add that "If dashboard module is already running, then you need to reload the dashboard module ..."

Reply:
It was my understanding that reloading the dashboard module is not sufficient when using the `ceph config-key set` command, as the entire manager instance needs to be restarted in order for this change to take effect. @jcsp can you confirm?

Reply:
You do need the daemon to restart, so doing a "ceph mgr fail mgr.<id>" is the natural way to do that.  However, disabling and re-enabling the module also works because ceph-mgr respawns when the list of enabled modules changes.